### ISSUE TITLE
Adding to screenshot to Anki card not working in Firefox

### ISSUE THREAD

**Issue by JasperSpruytte** [themoeway/yomitan#1121]
**Description**
I've configured my Anki setting to export a screenshot:
<img width="598" alt="image" src="https://github.com/themoeway/yomitan/assets/5804974/d713b8b9-7d42-4f8a-888c-d1a93e19aa42">
However, when I create the Anki card I get an error:
`chrome.tabs.captureVisibleTab is not a function`
Debug information from the console (Couldn't figure out how to properly copy it as text):
<img width="1275" alt="image" src="https://github.com/themoeway/yomitan/assets/5804974/09875900-328d-4eee-aa08-b9726ffec932">

**Browser version**
Firefox 127.0 (64-bit)

**Yomitan version**
Yomitan 24.6.17.0 (Downloaded from addons.mozilla.org)

**Exported settings file**
(If you think it might be relevant, create a settings export file using the _Export Settings_ button on the settings page and attach it to this issue.)

**Comment by Kuuuube** [comment]
I'm unable to replicate this.

Does this occur consistently or is this a "randomly" occurring issue? Are there any specific circumstances to trigger this?

Providing your settings may also help here.

**Comment by salixor** [comment]
Happens to me as well (Firefox 128.0.3, Yomitan 24.7.24.0). I am on Linux but I doubt it changes something.

Considering the displayed error talking about Chrome, I'd say it makes sense (unless Firefox uses the same API names).

**Comment by Kuuuube** [comment]
Theres `browser` and theres `chrome`. Firefox and chrome both generally support both. If your browser didnt have support for the `chrome` namespace for some reason yomitan would be much more broken than just not being able to take a screenshot.

Linux actually might matter here. Is it possible your version of firefox was compiled with nonstandard flags? Maybe theres something wacky going on in that way. Although I'm also running linux and still unable to reproduce this...

**Comment by salixor** [comment]
Maybe it would be a good thing, doesn't cost much to add a "Try to re-enable all files permission". Since in my case it was supposed to be enabled, but I had to re-enable for some odd reason.

**Comment by utzuro** [comment]
It happens on macOS too, and the disable-enable trick doesn't work for me.

**Comment by snah321** [comment]
Extension version: Yomitan 24.8.19.0
Platform: Windows
Browser: Firefox
Language: en-GB
User agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:129.0) Gecko/20100101 Firefox/129.0
AnkiConnect version: 6

Hello, I have the same error when trying to screenshot on firefox

**Comment by tenketsu2G** [comment]
Just migrated to Yomitan from Yomichan today, and ran into the same issue. Windows 11.

**Comment by Kuuuube** [comment]
> Had this issue too. I added `*://*` to the allowed origins (Enable Advanced Options), then it worked without errors.

Is anyone else able to verify this fix works for them? If it does we might be able to add it to docs somewhere.

**Comment by NathanYearout** [comment]
@salixor's fix worked for me. Appreciate the info here.

**Comment by cakiya** [comment]
got the same issue, but i first tried davdav's fix of adding `*://*`... i don't think its possible? how did you even add it?
<img width="906" height="596" alt="Image" src="https://github.com/user-attachments/assets/dd827bbe-3bec-4ab5-92b8-f47733803371" />

next i tried disabling and re-enabling yomitan and it worked for me...

yomitan from the official firefox addons listing 25.10.28.0, windows 11

**Comment by Kuuuube** [comment]
Yeah I got the same result attempting to test this back when I commented.

I recall another user resolving this by disabling and enabling Yomitan's permissions.
